tests/unit/test.vcxproj

```diff
@@ -93,7 +94,6 @@
       <SDLCheck>true</SDLCheck>
       <PreprocessorDefinitions>WIN32;_DEBUG;_CONSOLE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <ConformanceMode>true</ConformanceMode>
-      <RuntimeLibrary>MultiThreadedDebug</RuntimeLibrary>
```

Comment:
This change looks wrong, please revert.   Since PR #281, all debug builds should use MultiThreadedDebug not DLL which I think is the default.  (Line 128 is correct for x64)

Reply:
Should the default be MultiThreadedDebugDLL? MultiThreadedDebug causes the vc runtime library to be linked statically, and that causes an issue that I have mentioned in the other PR (#290)

Reply:
DLL is far harder to use for developers and we need to optimize the developer experience, hence the change in PR #281